sway/commands/resize.c

```diff
@@ -589,6 +589,11 @@ struct cmd_results *cmd_resize(int argc, char **argv) {
 	if (!current) {
 		return cmd_results_new(CMD_INVALID, "Cannot resize nothing");
 	}
+	if (current->scratchpad && !current->workspace) {
+		return cmd_results_new(CMD_FAILURE,
+				"Cannot resize a hidden scratchpad container");
+	}
+
```

Comment:
This breaks a floating terminal which was resized using a `for_window [app_id="quick_scratchpad"] mark quick, move scratchpad, resize set 1600 900` directive. The resize command is no longer applied (but worked before).